core/lib/asset-saver.js

```diff
@@ -420,14 +420,18 @@ async function saveLanternDebugTraces(pathWithBasename) {
  */
 async function saveAssets(artifacts, audits, pathWithBasename) {
   const allAssets = await prepareAssets(artifacts, audits);
-  const saveAll = allAssets.map(async (passAssets, index) => {
-    const devtoolsLogFilename = `${pathWithBasename}-${index}${devtoolsLogSuffix}`;
-    fs.writeFileSync(devtoolsLogFilename, JSON.stringify(passAssets.devtoolsLog, null, 2));
-    log.log('saveAssets', 'devtools log saved to disk: ' + devtoolsLogFilename);
-
-    const traceFilename = `${pathWithBasename}-${index}${traceSuffix}`;
-    await saveTrace(passAssets.traceData, traceFilename);
-    log.log('saveAssets', 'trace file streamed to disk: ' + traceFilename);
+  const saveAll = allAssets.map(async (assets, index) => {
+    if (assets.devtoolsLog) {
+      const devtoolsLogFilename = `${pathWithBasename}-${index}${devtoolsLogSuffix}`;
+      await saveDevtoolsLog(assets.devtoolsLog, devtoolsLogFilename);
```

Comment:
Quick callout, using `saveDevtoolsLog` instead of a generic `JSON.stringify` and `fs.writeFileSync`

Reply:
this is a readability change, but in what direction is subjective. seems fine.